app/services/container_dashboard_service.rb

```diff
@@ -120,39 +122,35 @@ def heatmaps
     node_ids = @ems.container_nodes if @ems
     metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
 
-    {
-      :nodeCpuUsage => metrics.collect { |m|
-        avg_cpu = m.cpu_usage_rate_average.round(2)
-        {
-          :id      => m.resource_id,
-          :tooltip => "#{m.resource.name} - #{avg_cpu}%",
-          :value   => avg_cpu / 100.0 # 1% should be 0.01
-        }
-      },
-      :nodeMemoryUsage => metrics.collect { |m|
-        avg_mem = m.mem_usage_absolute_average.round(2)
-        {
-          :id      => m.resource_id,
-          :tooltip => "#{m.resource.name} - #{avg_mem}%",
-          :value   => avg_mem / 100.0 # 1% should be 0.01
+    if metrics.any?
+      {
+        :nodeCpuUsage => metrics.collect { |m|
```

Comment:
@zeari for multiple lines you should use `do` in general. Is it not working in this specific case?